ext/mysql2/client.c

```diff
@@ -1311,9 +1313,11 @@ static VALUE set_ssl_options(VALUE self, VALUE key, VALUE cert, VALUE ca, VALUE
   return self;
 }
 
+#if defined(MYSQL_SECURE_AUTH)
```

Comment:
Make this method a no-op by putting the `#ifdef` inside the method definition. Please add a comment, something like:

```
/* This option was deprecated in MySQL 5.x and removed in MySQL 8.0 */

Reply:
Thanks for the review. Changed ifdef location inside the method.